### PR TITLE
[DRTV] Fix extractor to allow for new URL scheme for radio episodes

### PR DESCRIPTION
**Pull request by vertan**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

---

### Description of your *pull request* and other information
DR have changed their URL scheme for radio episodes from `https://www.dr.dk/radio/` to `https://www.dr.dk/lyd/`, so this PR changes to allowed URL. There's also a couple of fixes to old tests in there to make them all green.

### CODE REVIEW

File: yt_dlp/extractor/drtv.py [hunk 1]
```diff
@@ -2,28 +2,27 @@
...
     float_or_none,
+    int_or_none,
     mimetype2ext,
     str_or_none,
     try_get,
     unified_timestamp,
     update_url_query,
     url_or_none,
 )
+from .common import InfoExtractor
```
Comment (pukkandan) [inline yt_dlp/extractor/drtv.py:18]: move this back above aes

**Review by pukkandan — CHANGES_REQUESTED** [review]

**Comment by vertan** [comment]
Thanks for the input @pukkandan, I have fixed your comments

File: yt_dlp/extractor/drtv.py [hunk 2]
```diff
@@ -71,32 +71,44 @@ class DRTVIE(InfoExtractor):
...
+            'release_year': 2017,
+            'season_id': "urn:dr:mu:bundle:5afc03ad6187a4065ca5fd35",
+            'season_number': 1,
+            'season': "Historien om Danmark",
+            'series': "Historien om Danmark"
         },
         'params': {
             'skip_download': True,
         },
+        'expected_warnings': ['Ignoring subtitle tracks found in the HLS manifest']
```
Comment (pukkandan) [inline yt_dlp/extractor/drtv.py:91]:
```suggestion
```

File: yt_dlp/extractor/drtv.py [hunk 3]
```diff
@@ -71,32 +71,44 @@ class DRTVIE(InfoExtractor):
...
+            'timestamp': 1603188600,
+            'upload_date': '20201020',
             'duration': 2576.6,
+            'season': 'Bonderøven 2019',
+            'season_id': 'urn:dr:mu:bundle:5c201667a11fa01ca4528ce5',
+            'release_year': 2019,
+            'season_number': 2019,
+            'series': 'Frank & Kastaniegaarden'
         },
+        'expected_warnings': ['Ignoring subtitle tracks found in the HLS manifest'],
```
Comment (pukkandan) [inline yt_dlp/extractor/drtv.py:111]:
```suggestion
```

**Review by pukkandan — CHANGES_REQUESTED** [review]
the warnings are an indication of an issue in the code. You don't need to fix it rn, but simply suppressing it is not good idea

File: yt_dlp/extractor/drtv.py [hunk 4]
```diff
@@ -109,6 +121,24 @@ class DRTVIE(InfoExtractor):
     }, {
         'url': 'https://www.dr.dk/drtv/program/jagten_220924',
         'only_matching': True,
+    }, {
+        'url': 'https://www.dr.dk/lyd/p4aarhus/regionale-nyheder-ar4/regionale-nyheder-2022-05-05-12-30-3',
+        'info_dict': {
+            'id': "urn:dr:mu:programcard:6265cb2571401424d0360113",
```
Comment (pukkandan) [inline yt_dlp/extractor/drtv.py:127]:
```suggestion
            'id': 'urn:dr:mu:programcard:6265cb2571401424d0360113',
```

File: yt_dlp/extractor/drtv.py [hunk 5]
```diff
@@ -71,32 +71,44 @@ class DRTVIE(InfoExtractor):
...
             'ext': 'mp4',
             'title': 'Historien om Danmark: Stenalder',
             'description': 'md5:8c66dcbc1669bbc6f873879880f37f2a',
             'timestamp': 1546628400,
             'upload_date': '20190104',
-            'duration': 3502.56,
+            'duration': 3504.618,
             'formats': 'mincount:20',
+            'release_year': 2017,
+            'season_id': "urn:dr:mu:bundle:5afc03ad6187a4065ca5fd35",
```
Comment (pukkandan) [inline yt_dlp/extractor/drtv.py:83]:
```suggestion
            'season_id': 'urn:dr:mu:bundle:5afc03ad6187a4065ca5fd35',
```

File: yt_dlp/extractor/drtv.py [hunk 6]
```diff
@@ -71,31 +71,41 @@ class DRTVIE(InfoExtractor):
...
             'timestamp': 1546628400,
             'upload_date': '20190104',
-            'duration': 3502.56,
+            'duration': 3504.618,
             'formats': 'mincount:20',
+            'release_year': 2017,
+            'season_id': 'urn:dr:mu:bundle:5afc03ad6187a4065ca5fd35',
+            'season_number': 1,
+            'season': "Historien om Danmark",
+            'series': "Historien om Danmark"
```
Comment (pukkandan) [inline yt_dlp/extractor/drtv.py:86]:
```suggestion
            'season': 'Historien om Danmark',
            'series': 'Historien om Danmark',
```

**Comment by vertan** [comment]
Thanks for the comments and fixes! First time committing to this repo, will try to think of that next time

**Pull request merged**